etl/steps/data/garden/un/2024-07-25/refugee_data.meta.yml

```diff
@@ -172,13 +172,13 @@ tables:
       others_of_concern_asylum:
         title: Other persons of concern by country of residence
         description_short: |-
-          [Other persons of concern](#dod:unhcr-concern) to the UNHCR in this country. This measures the total number of persons; not the number per year.
+          [Other persons of concern](#dod:unhcr-concern) to the UNHCR in this country. This measures the total number of persons of concern living in this country.
```

Comment:
Perhaps worth adding 'not the number of new persons of concern arriving this year', for completeness? Also for 'Stateless persons' and 'Other people in need of international protection'.